CMakeLists.txt

```diff
@@ -1960,6 +1960,12 @@ if(HPX_WITH_VIM_YCM)
   hpx_info("VIM YouCompleteMe: run 'make configure_ycm' to copy config file to source directory and enable support in YCM. To enable automatic loading of configure file, add to your .vimrc option: \"let g:ycm_extra_conf_globlist = ['${CMAKE_SOURCE_DIR}/*']\"")
 endif()
 
+################################################################################
+# Add rpm packaging
+if(HPX_WITH_RPM)
```

Comment:
`HPX_WITH_RPM` should probably still be added with `hpx_option` in the main `CMakeLists.txt`.

Reply:
Add the hpx_option in the latest commit